src/runtime_src/core/include/xrt/xrt_bo.h

```diff
@@ -111,18 +111,22 @@ class bo
    *  Create a BO for peer-to-peer use
    * @var svm
    *  Create a BO for SVM (supported on specific platforms only)
-   *
+   * @var device_memory
+   *  Create a BO from device memory, which is a separate pool of
```

Comment:
We should qualify this crisply so end users can understand this.
Create a BO from a special memory pool dedicated to the device.  For AMD Ryzen NPU
this memory is allocated from the host memory carveout pool.

Reply:
Please don't use carveout.  How  about "allocated from a reserved host memory pool"